CMakeLists.txt

```diff
@@ -245,7 +245,7 @@ endif()
 
 # Boost
 if(NOT DEFINED Boost_INCLUDE_DIR OR NOT DEFINED Boost_LIBRARIES)
-	FIND_PACKAGE( Boost COMPONENTS system thread date_time chrono random)
+	FIND_PACKAGE(Boost REQUIRED COMPONENTS system thread date_time chrono random)
```

Comment:
technically, thread and date_time aren't required. boost.asio has been pulling in date_time headers for a while, but I believe there's a way to make it stop doing that (and it's possible that now that I use the chrono based primitives it won't anymore)